CRM/Admin/Form/ScheduleReminders.php

```diff
@@ -457,8 +463,8 @@ public function postProcess() {
         );
       }
 
-      if ($this->_context == 'event' && $this->_compId) {
-        $url = CRM_Utils_System::url('civicrm/event/manage/reminder', "reset=1&action=browse&id={$this->_compId}&component={$this->_context}&setTab=1");
+      if ($this->getContext() == 'event' && $this->$this->getComponentID()) {
```

Comment:
Seems like a typo - `$this->$this->getComponentID();`

Reply:
arg - fixed - thanks